src/SourceBrowser/src/SourceIndexServer/Program.cs

```diff
@@ -10,11 +11,19 @@ public static void Main(string[] args)
             BuildWebHost(args).Run();
         }
 
+        public static ILogger Logger { get; set; }
+
         public static IHost BuildWebHost(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(
                     builder => { builder
                         .UseStartup<Startup>(); })
+                .ConfigureLogging(logging =>
+                        {
+                            logging.ClearProviders();

```

Comment:
Why do we need to clear the providers here?

Reply:
It just seems to be the common idiom to clear what's there already if overriding defaults. Presumably an exception if trying to add a duplicate of a default.